test/pythonml/test_model_flavors.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import importlib
+import os
+import shutil
+import unittest
+from parameterized import parameterized
+from fixtures.unittest_baseclass import BaseUnitTest
```

Comment:
The base-class isn't really a fixture, we should move it out of the fixtures folder